test/data/case5_mld_strgonly.m

```diff
@@ -0,0 +1,59 @@
+% used in tests of,
```

Comment:
This file name should be `case5_mld_strg_only.m`.  Also why "only"?

Reply:
this model has not generators, only storage.  What is an appropriate name?

Reply:
I didn't notice that the status was off for all of the generators.  This name is good.